nautobot/extras/models/approvals.py

```diff
@@ -151,6 +162,9 @@ def __str__(self):
         """Stringify instance."""
         return f"{self.approval_workflow.name}: {self.object_under_review} ({self.current_state})"
 
+    def get_current_state_class(self):
```

Comment:
Perhaps `get_current_state_css_class` or `@property current_state_css_class`?

Reply:
This is for rendering for the [ChoicesFieldColumn](https://github.com/nautobot/nautobot/blob/next/nautobot/core/tables.py#L451) as it is looking for this method name specifically.